src/components/Person/Person.jsx

```diff
@@ -1 +1,11 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    <p className="Person__partner">
+      {person.isMarried
+        ? (`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`)
+        : (`I am not married`)}
+    </p>
+  </section>
```

Comment:
use destructuring + fix code style, add some spaces, put properties on the new line + get rid of ternary inside of ternary, create variable for it